examples/Quotes.hs

```diff
@@ -0,0 +1,10 @@
+module Main where
+
+import Criterion
+import Criterion.Main
+
+main :: IO ()
+main = defaultMain
+    [ env (return ()) $
+       \ ~() -> bgroup "\"oops\"" [bench "dummy" $ nf id ()]
```

Comment:
Can you also add a benchmark that uses single quotes instead of double quotes? That way, we can test both #202 and #235 simultaneously.

Reply:
Thank you for the suggest, I have now added a similar benchmark that uses single quotes and tested that the generated JSON and HTML looks correct.